### ISSUE TITLE
Add a contact section in the navbar

### ISSUE THREAD

**Issue by sarnava1** [fossasia/meilix-generator#280]
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

**Actual Behaviour**
We should have a contact section which should appear on clicking the menu icon in the navbar as project susper has.

![contactinsusper](https://user-images.githubusercontent.com/25319565/46951971-2db8c600-d0a7-11e8-9d66-1cd53c520d49.png)

**Would you like to work on the issue?**
YES if mentors and other contributors like the idea.

**Comment by sarnava1** [comment]
@meets2tarun and @tabesin presently we have blogs, code and bug under menus in the navbar. When I was checking out susper I saw about the contact section which they have. I though of having it in our meilix project too subjected to your approval.

**Comment by meets2tarun** [comment]
Except susper, FOSSASIA has also other web app, please confirm in those too.

Thanks

**Comment by meets2tarun** [comment]
<img width="1680" alt="screenshot 2018-10-18 at 22 02 03" src="https://user-images.githubusercontent.com/15213246/47169572-8abfb080-d321-11e8-9088-27003de314af.png">

**Comment by sarnava1** [comment]
We do not find a contact us section here. Let me search for more web apps.

**Comment by tabesin** [comment]
The question is just if the contact section is needed at this stage. In particular as the core functionality does not work yet. If you want to do it, just proceed. But we shouldn't then have "like 20" pull request to further improve the contacts page because that's lipstick on a pig.

The GUI as it is is good enough. Now it is important to get the UI functionality in and then ensure that the Meilix backend does what we (or the app users triggering build) want it to do and suceeds to deliver the custom Meilix ISO.

**Comment by sarnava1** [comment]
Yes @tabesin it does not require much attention right now. Let this be open for discussions and ages on after the main work is do e we can focus on this.

**Comment by vijay938** [comment]
I want to work on this issue
